api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -44,6 +44,8 @@ service StellarStationService {
   // The request will be closed with an `INVALID_ARGUMENT` status if `ground_station_id`,
   // `aos_after`, or `aos_before` are missing, or the duration between the two times is longer than
   // 31 days.
+  //
+  // Deprecated - use GroundStationService.ListGroundStationPlans
```

Comment:
Need to fix as well: GroundStationService.ListPlans

Reply:
Fixed